.travis.yml

```diff
@@ -19,5 +15,5 @@ script:
   - pip install numpy
   - sbt test
 after_success:
-  - 'if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./publish.sh; fi'
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [ -n "$TRAVIS_TAG" ]; then bash ./publish.sh; fi'
+  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ] && [ "$TRAVIS_BRANCH" = "master" ]; then bash ./publish.sh; fi'
```

Comment:
why  do we need `publish` at all? (I;m not good at CI )

Reply:
Just running on `master` and tags - we won't need that since in our fork all PRs are ultimately merged to `develop`; `master` branch in turn does NOT exist in this fork

Reply:
So `publish` stuff is only ever being run for the main repo, and not our fork